apps/landing/components/Terms/Content.tsx

```diff
@@ -0,0 +1,339 @@
+// TODO: convert this component to server component once the Typography task is done
+"use client";
+
+import React from "react";
+import cn from "classnames";
+import { useFormatMessage } from "@/hooks/intl";
+import { Typography } from "@litespace/ui/Typography";
+
+import Facebook from "@litespace/assets/Facebook";
+import Youtube from "@litespace/assets/Youtube";
+import Insta from "@litespace/assets/Instagram";
+import Linkedin from "@litespace/assets/Linkedin";
+import Whatsapp from "@litespace/assets/Whatsapp";
+import Link from "next/link";
+
+const Content: React.FC = () => {
+  const intl = useFormatMessage();
+  return (
+    <div
+      className={cn(
+        "flex flex-col py-14 sm:py-20 md:py-20 px-4 md:px-8 lg:px-[108px] gap-14 sm:gap-20",
+        "max-w-screen-3xl mx-auto w-full"
+      )}
+    >
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/policies-updates/title")}
+        </Typography>
+
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/policies-updates/content/1")}
+        </Typography>
+
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/policies-updates/content/2")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/account-register/title")}
+        </Typography>
+
+        <BulletList
```

Comment:
I believe this component can be shared between pages. Let's move it into `components/common/`